### ISSUE TITLE
`initialize': uninitialized constant DockerSync::DockerComposeSession

### ISSUE THREAD

**Issue by dswtpvtltd** [EugenMayer/docker-sync#817]
### Error/Feature Requestion/Docs
?

WARNING: The requested image's platform (linux/amd64) does not match the detected host platform (linux/arm64/v8) and no specific platform was requested
          ok  Synced /Volumes/DATA/src/web-server-sync/
     success  Rsync server started
          ok  Synced /Volumes/DATA/src/web-server-sync/src/web-server-sync/
     success  Starting to watch /Volumes/DATA/src/web-server-sync/src/web-server-sync/ - Press CTRL-C to stop
          ok  Found implicit docker-compose-dev.yml and using it from /Volumes/DATA/src/web-server-sync/docker-compose-dev.yml
/Users/abc/.gem/ruby/2.6.0/gems/docker-sync-1.0.0/lib/docker-sync/compose.rb:32:in `initialize': uninitialized constant DockerSync::DockerComposeSession (NameError)
	from /Users/abc/.gem/ruby/2.6.0/gems/docker-sync-1.0.0/tasks/stack/stack.thor:47:in `new'
	from /Users/abc/.gem/ruby/2.6.0/gems/docker-sync-1.0.0/tasks/stack/stack.thor:47:in `start'
	from /Users/abc/.gem/ruby/2.6.0/gems/thor-1.2.1/lib/thor/command.rb:27:in `run'
	from /Users/abc/.gem/ruby/2.6.0/gems/thor-1.2.1/lib/thor/invocation.rb:127:in `invoke_command'
	from /Users/abc/.gem/ruby/2.6.0/gems/thor-1.2.1/lib/thor.rb:392:in `dispatch'
	from /Users/abc/.gem/ruby/2.6.0/gems/thor-1.2.1/lib/thor/base.rb:485:in `start'
	from /Users/abc/.gem/ruby/2.6.0/gems/docker-sync-1.0.0/bin/docker-sync-stack:15:in `<top (required)>'
	from /usr/local/bin/docker-sync-stack:23:in `load'
	from /usr/local/bin/docker-sync-stack:23:in `<main>'
make: *** [up] Error 1

### Docker Driver
? (d4m or docker-machine+vbox/fusion)

### Sync strategy
? (default/native_osx/unison/rsync)
rsync

### your docker-sync.yml
version: "2"

syncs:
  web-server-sync:
    src: "./src"
    sync_strategy: "rsync"
    sync_host_port: ${RSYNC_PORT}
    sync_userid: 34000
    cli_mode: "docker-sync"
    sync_excludes:
      [
        "/.composer", 
       "/generated", 
       "/pub/static", 
       "/vendor", 
       "/var", 
       ".*.sw?"
     ]

### OS
? (OSX Version/Linux)
macOs Monterey 12.5.1

**Comment by apollon-nishikawa-irsc** [comment]
same here.
After updated to docker-sync 1.0.0, I can't start with command `docker-sync-stack start`.
But I can run, `docker-sync start`, and `docker-compose up` without any problem.

### Error/Feature Requestion/Docs
```
Traceback (most recent call last):
        9: from ~/.rbenv/versions/2.7.6/bin/docker-sync-stack:23:in `<main>'
        8: from ~/.rbenv/versions/2.7.6/bin/docker-sync-stack:23:in `load'
        7: from ~/.rbenv/versions/2.7.6/lib/ruby/gems/2.7.0/gems/docker-sync-1.0.0/bin/docker-sync-stack:15:in `<top (required)>'
        6: from ~/.rbenv/versions/2.7.6/lib/ruby/gems/2.7.0/gems/thor-1.2.1/lib/thor/base.rb:485:in `start'
        5: from ~/.rbenv/versions/2.7.6/lib/ruby/gems/2.7.0/gems/thor-1.2.1/lib/thor.rb:392:in `dispatch'
        4: from ~/.rbenv/versions/2.7.6/lib/ruby/gems/2.7.0/gems/thor-1.2.1/lib/thor/invocation.rb:127:in `invoke_command'
        3: from ~/.rbenv/versions/2.7.6/lib/ruby/gems/2.7.0/gems/thor-1.2.1/lib/thor/command.rb:27:in `run'
        2: from ~/.rbenv/versions/2.7.6/lib/ruby/gems/2.7.0/gems/docker-sync-1.0.0/tasks/stack/stack.thor:47:in `start'
        1: from ~/.rbenv/versions/2.7.6/lib/ruby/gems/2.7.0/gems/docker-sync-1.0.0/tasks/stack/stack.thor:47:in `new'
~/.rbenv/versions/2.7.6/lib/ruby/gems/2.7.0/gems/docker-sync-1.0.0/lib/docker-sync/compose.rb:32:in `initialize': uninitialized constant DockerSync::DockerComposeSession (NameError)
```

### your docker-sync.yml
```
version: '2'
syncs:
  sync-data:
    notify_terminal: true
    sync_host_ip: '127.0.0.1'
    sync_host_port: 10871
    src: '.'
    sync_strategy: 'native_osx'
    sync_excludes: ['vendor/bundle', '.bundle', 'tmp/cache', '.idea', '.git', '.gitignore', '.DS_Store', 'public/assets/javascripts', 'public/assets/stylesheets', 'node_modules']
    sync_excludes_type: 'Path'

```
### OS
macOS Monterey
12.3.1

**Comment by mateussantiago** [comment]
I see the same issue as well. I've rolled back to the 0.7.2 version to get it to work again.

**Comment by EugenMayer** [comment]
Fixed with 1.0.1, thank you for repoting
